src/main/kotlin/blackjack/dealer/Dealer.kt

```diff
@@ -1,17 +1,24 @@
 package blackjack.dealer
 
+import blackjack.player.Hand
 import blackjack.player.Player
 
-class Dealer {
+class Dealer(
+    name: String = "딜러",
+    hand: Hand = Hand()
+) : Player(name, hand) {
```

Comment:
`Dealer`의 경우 이름이 딜러로 항상 정해져있으니 다음과 같이 구현해볼 수 있을 것 같아요 :) 
또한 딜러로 이름이 고정되어있는데 외부에서 딜러가아닌 다른 이름을 변경할 수 있을 것 같아요
```suggestion
class Dealer(
    hand: Hand = Hand()
) : Player("딜러", hand) {
```